go/host/rpc/clientapi/client_api_eth.go

```diff
@@ -49,11 +49,24 @@ func (api *EthereumAPI) GetBalance(_ context.Context, encryptedParams common.Enc
 
 // GetBlockByNumber returns the rollup with the given height as a block. No transactions are included.
 func (api *EthereumAPI) GetBlockByNumber(_ context.Context, number rpc.BlockNumber, _ bool) (map[string]interface{}, error) {
+	var rollupHeaderWithHashes *common.HeaderWithTxHashes
+
+	switch number {
```

Comment:
just out of curiosity.
Is this something that geth does as well?

Reply:
Yep!
https://github.com/ethereum/go-ethereum/blob/c6dcd018d25998201eff1e1e264c9080847d92df/rpc/types.go#L77-L110